_posts/zh/newsletters/2023-05-31-newsletter .md

```diff
@@ -0,0 +1,129 @@
+  Murch 解释说，对于不使用裸多重签名的多重签名设置，除非以前使用相同的多重签名输出脚本，否则需要所有公钥才能使用。他指出，“多重签名钱包的备份策略必须既保留私钥，又保留输出的条件脚本”，并建议将[描述符][topic descriptors]作为备份条件脚本的方法。{% assign timestamp="1:23:50" %}
+
+## 版本和候选版本
+
+*热门的比特币基础设施项目的新版本和候选版本。请考虑升级到新版本或帮助测试候选版本。*
+
+- [Bitcoin Core 25.0][]是 Bitcoin Core 下一个主要版本。该版本添加了一个新的`scanblocks`RPC，简化了`bitcoin-cli`的使用，为`finalizepsbt`RPC 添加了[miniscript][topic miniscript]支持，使用`blocksonly`配置选项减少了默认内存使用，并在启用[致密区块过滤器][topic compact block filters]时加快了钱包重新扫描的速度——以及许多其他新功能、性能改进和错误修复。有关详细信息，见[版本文档][bcc rn]。{% assign timestamp="1:27:05" %}
+
+## 重大的代码和文档变更：
+
+*本周的重大变更有：[Bitcoin Core][bitcoin core repo]、[Core Lightning][core lightning repo]、[Eclair][eclair repo]、[LDK][ldk repo]、[LND][lnd repo]、[libsecp256k1][libsecp256k1 repo]、[Hardware Wallet Interface (HWI)][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、[BTCPay Server][btcpay server repo]、[BDK][bdk repo]、[Bitcoin Improvement Proposals (BIPs)][bips repo]、[Lightning BOLTs][bolts repo]和[Bitcoin Inquisition][bitcoin inquisition repo]。*
+
+- [Bitcoin Core #27469][]在使用一个或多个钱包时加速了初始区块下载（IBD）。通过此更改，如果区块是在钱包的出生日期（钱包中记录的创建日期）之后开采的，则只会扫描与特定钱包匹配的交易。 {% assign timestamp="1:30:27" %}
+
+- [Bitcoin Core #27626][]允许要求我们的节点在高带宽模式下提供[致密区块中继][topic compact block relay]的对等方从我们向他们发布的最新块发出最多三个交易请求。即使我们最初没有向它们提供致密区块，我们的节点也会响应请求。这允许从其他对等方之一接收致密区块的对等节点向我们请求任何丢失的交易，如果另一个对等节点没有响应，这可以提供帮助。这可以帮助我们的对等方更快地验证区块，这也可以帮助他们在时间关键型功能（如挖矿）中更快地使用它。{% assign timestamp="1:32:40" %}
+
+- [Bitcoin Core #25796][]添加了一个新的`descriptorprocesspsbt`，可用于更新[PSBT][topic psbt]，这些信息将有助于以后签名或最终确定。提供给 RPC 的[描述符][topic descriptors]将用于从交易池和 UTXO 集中检索信息(并且，如果可用，则在使用`txindex`配置选项启动节点时完成已确认的事务)。然后，检索到的信息将用于填写 PSBT。{% assign timestamp="1:35:24" %}
+
+- [Eclair #2668][]阻止 Eclair 试图支付比成功解析该 HTLC 所获得的价值更多的费用来申领链上 [HTLC][topic htlc]。{% assign timestamp="1:37:17" %}
+
+- [Eclair #2666][]允许接收 [HTLC][topic htlc] 的远程节点接受它，即使接受它需要支付的交易费用会将节点的余额减少到最低通道储备以下。通道储备的存在是为了确保节点在试图关闭处于过时状态的通道时至少会损失少量资金，从而阻止他们盗窃的企图。但是，如果远程节点接受 HTLC，一旦成功，HTLC 将向他们付款，那么无论如何，他们将面临比储备金更多的风险。如果不成功，他们的余额将恢复到之前的金额，该金额将高于储备金。
+
+    这是对资金滞留问题的缓解措施，资金滞留问题发生在一笔付款的手续费承担方需要支付比其当前可用余额更多的价值时，即使他们可能是接收付款的一方，也会发生这种情况。有关此问题的先前讨论，见[周报 #85][news85 stuck funds]. {% assign timestamp="1:39:20" %}
+
+- [BTCPay Server 97e7e][]开始为[payjoin][topic payjoin]支付设置[BIP78][]`minfeerate`(最小费率)参数。另请参阅导致此提交的[错误报告][btcpay server #4689]。{% assign timestamp="1:41:56" %}
+
+- [BIPs #1446][]对比特币相关协议的[schnorr 签名][topic schnorr signatures]的[BIP340][]规范进行了小改动和一些补充。此更改允许对消息进行任意长度的签名；以前版本的 BIP 要求消息正好为 32 个字节。描述对 Libsecp256k1 库的相关更改请见 [周报 #157][news157 libsecp]。此更改对共识应用程序中 BIP340 的使用没有影响，因为与[taproot][topic taproot]和[tapscript][topic tapscript] (分别为，BIPs[341][bip341] and [342][bip342])一起使用的签名使用 32 字节消息。
+
+    新增内容描述了如何有效地使用任意长度的消息，推荐了如何使用散列标记前缀，并提供了在不同域中使用相同的密钥（例如签署交易或签署纯文本消息）时提高安全性的建议。 {% assign timestamp="1:43:15" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="27469,27626,25796,2668,2666,4689,1446" %}
+[policy series]: /en/blog/waiting-for-confirmation/
```

Comment:
@hulatown would you mind creating a zh version of the policy series similar to the fr version here: https://github.com/bitcoinops/bitcoinops.github.io/pull/1147/files#diff-d5483c9b332b1fe2721efc5852c20afac39aaa0dfcbd390d61d346d296706149

Reply:
Added! Sorry for late reply